Code/GraphMol/MolStandardize/Validate.h

```diff
@@ -84,7 +84,8 @@ class RDKIT_MOLSTANDARDIZE_EXPORT MolVSValidations {
 
 //! The NoAtomValidation class throws an error if no atoms are present in the
 // molecule.
-class RDKIT_MOLSTANDARDIZE_EXPORT NoAtomValidation : public MolVSValidations {
+class RDKIT_MOLSTANDARDIZE_EXPORT NoAtomValidation final
```

Comment:
What is the rational to use final here?

Reply:
The warning here was:
```
/Users/travis/conda/include/boost/smart_ptr/make_shared_object.hpp:59:13: warning: destructor called on non-final 'RDKit::MolStandardize::NoAtomValidation' that has virtual functions but non-virtual destructor [-Wdelete-non-virtual-dtor]
            p->~T();
            ^
```

Reply:
Perhaps another solution is to give MolVSValidations a virtual destructor.

Reply:
I like this. It makes it clear that these classes shouldn't be derived from  (and there's not really any reason to do so).